scout/supporting_data/config_schema.yml

```diff
@@ -0,0 +1,243 @@
+        default: false
+        description: Disable secondary lighting.
+
+      floor_start:
+        type: ["integer", "null"]
+        default: null
+        description: The starting floor.
+
+      pkg_env_costs:
+        type: ["string", "null"]
+        enum: [include HVAC, exclude HVAC, null]
+        description: Define what measure data should be written out for inclusion in measure competition. `include HVAC` will prepare HVAC-only versions of all HVAC/envelope packages for measure competition, `exclude HVAC` will exclude these HVAC-only measures from measure competition.
+
+      exog_hp_rates:
+        type: object
+        properties:
+          exog_hp_rate_scenario:
+            type: ["string", "null"]
+            enum: [conservative, optimistic, aggressive, most aggressive, null]
+            description: Guidehouse E3HP conversion scenario to use for exogenous HP rates.
+          switch_all_retrofit_hp:
+            type: boolean
+            description: Assume all retrofits convert to heat pumps, otherwise, retrofits are subject to the same external heat pump conversion rates assumed for new/replacement decisions. Only applicable if `exog_hp_rate_scenario` and `retrofit_type` are defined.
+        if:
+          properties:
+            exog_hp_rate_scenario:
+              enum: [conservative, optimistic, aggressive, most aggressive]
+        then:
+          required: [exog_hp_rate_scenario]
+
+      alt_ref_carb:
+        type: boolean
+        default: false
+        description: set the baseline electricity emissions intensities to be consistent with the Standard Scenarios Mid Case (rather than AEO).
+
+      grid_decarb:
+        type: object
+        required: [grid_decarb_level, grid_assesment_timing]
+        properties:
+          grid_decarb_level:
+            type: ["string", "null"]
+            enum: ["0.8", full, null]
+            description: Enable grid decarbonization - `0.8` represents 80%% reduced grid emissions from current levels by 2050, and `full` represents full grid decarbonization by 2035.
+          grid_assesment_timing:
+            type: ["string", "null"]
+            enum: [before, after, null]
+            description: When to assess avoided emissions and costs from non-fuel switching measures relative to the additional grid decarbonization.
+
+      adopt_scn_restrict:
+        type: ["string", "null"]
+        enum: [Max adoption potential, Technical potential, null]
+        description: Specify adoption scenario restrictions.
+
+      retrofits:
+        type: object
+        required: [retrofit_type]
+        properties:
+          retrofit_type:
+            type: ["string", "null"]
+            enum: [constant, increasing, null]
+            description: Type of retrofit. `constant` assumes component-based early retrofit rates that do not change over time, `increasing` assumes rates that increase over time.
+          retrofit_multiplier:
+            type: ["number", "null"]
```

Comment:
Is "number" a type? I would expect the non-integer numeric type to be "float."

Reply:
Number is basically a float: https://json-schema.org/understanding-json-schema/reference/numeric#number